mne/epochs.py

```diff
@@ -3678,6 +3678,9 @@ def make_fixed_length_epochs(raw, duration=1., preload=False,
         Raw data to divide into segments.
     duration : float
         Duration of each epoch in seconds. Defaults to 1.
+    overlap : float
+        The overlap between epochs. Must be ``0 <= overlap < duration``.
+        Default is 0.
```

Comment:
you need to add the `versionadded` directive to document when this parameter appeared in a release.

Reply:
Thanks, added